### ISSUE TITLE
Inside of module rpc y_op_invoke code - add data to reply

### ISSUE THREAD

**Issue by ronniebrash** [OpenClovis/OpenYuma#26]
Hello,

How do I add data to the RPC's payload which will be sent back to the yangcli?  I have lets say an arbitrary buffer.

**Comment by hoangle** [comment]
It should be defined at 'output' of RPC statements: refer to [https://tools.ietf.org/html/rfc6020#section-4.2.9]

**Comment by ronniebrash** [comment]
Yes, I know that to be the case, however, getting the buffer inside of the rpc-payload is not trivial from the documentation.

So far I have:
* Yang model to be used by yangdump to make a stub
* Filled in stubs and used a specific invoke function
* Set and created an output callback
* Copied the buffer into the payload via ses' specifically in a way which causes the validation to break. It mentions xmlany.

In yangcli, it looks like:

Reply{
Data '...'
}

Is there a better way?
My data is raw XML and I would like it to be a payload. I noticed that yang doesn't have an output type for that and the ses* code parses it.

**Comment by hoangle** [comment]
My recommend is to use dummy session and agt_val_parse_nc to parse buffer (i.e xml data) into val value.
Take a look at function agt_val_parse_test on agt_val_parse.c
